qa/workunits/fs/full/subvolume_clone.sh

```diff
@@ -7,8 +7,8 @@ set -ex
 # Hence the subsequent subvolume commands on the clone fails with
 # 'MetadataMgrException: -2 (section 'GLOBAL' does not exist)' traceback.
 
-# The osd is of the size 1GB. The full-ratios are set so that osd is treated full
-# at around 600MB. The subvolume is created and 100MB is written.
+# The osd is of the size 2GiB. The full-ratios are set so that osd is treated full
+# at around 1.2GB. The subvolume is created and 200MB is written.
```

Comment:
mixed usage of giga/gibi look a bit odd to me

Reply:
Yeah, but that's how the test case is written. The `dd` command used writes in MB/GB but the bluefs allocation is done GiB.